### PR TITLE
14825 remove optional first name in search

### PR DESCRIPTION
**Pull request by owillborn**
*Issue #:* /bcgov/entity#14825

*Description of changes:*
-Removed "(Optional)" text for Debtor Name search First name text field for staff accounts
-Fixed icon alignment for ticket #15029
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/search/SearchBar.vue [hunk 1]
```diff
@@ -87,7 +87,7 @@
                   filled
                   :hint="searchHintFirst"
                   persistent-hint
-                  :label="optionFirst"
+                  label="First Name"
```
Comment (dimak1) [inline ppr-ui/src/components/search/SearchBar.vue]: Just to confirm, the _Optional_ is removed from all roles? I remember it was added recently, and now removed?

Comment (owillborn) [reply ppr-ui/src/components/search/SearchBar.vue]: Ticket called for it to be removed from staff accounts, it was already removed from the other roles. The validation for staff was already treating it as a required field, but the "(Optional)" label was still there...

Comment (owillborn) [reply ppr-ui/src/components/search/SearchBar.vue]: Will double check though, does seem a bit odd

Comment (owillborn) [reply ppr-ui/src/components/search/SearchBar.vue]: @dimak1 Confirmed with Divya, the only case in which it should have the optional first name field is for a staff account when doing an MHR Owner Name search. Will refactor

Comment (dimak1) [reply ppr-ui/src/components/search/SearchBar.vue]: Great, thank you!

**Review by ChasBoucher — APPROVED** [review]
Looks Good

**Review by cameron-eyds — APPROVED** [review]

**Review by dimak1 — APPROVED** [review]

**Review by doug-lovett — APPROVED** [review]

**Review by ChasBoucher — APPROVED** [review]
Looks Good

**Pull request merged**